apps/lenra/lib/lenra/apps.ex

```diff
@@ -187,6 +188,40 @@ defmodule Lenra.Apps do
     Repo.fetch_by(Environment, id: main_env.environment_id)
   end
 
+  ###############
+  # Env Secrets #
+  ###############
+
+  def all_env_secrets_for_env(env_id) do
+    Repo.all(from(s in EnvSecret, where: s.environment_id == ^env_id))
+  end
+
+  def get_env_secret(secret_id) do
+    Repo.get(EnvSecret, secret_id)
+  end
+
+  def fetch_env_secret(secret_id) do
+    Repo.fetch(EnvSecret, secret_id)
+  end
+
+  def create_env_secret(env_id, key, params) do
+    Ecto.Multi.new()
+    |> Ecto.Multi.insert(:inserted_env_secret, EnvSecret.new(env_id, key, params))
+    |> Repo.transaction()
+  end
+
+  def update_env_secret(env_secret, params) do
+    Ecto.Multi.new()
+    |> Ecto.Multi.update(:updated_env_secret, EnvSecret.update(env_secret, params))
+    |> Repo.transaction()
+  end
+
+  def delete_env_secret(env_secret, params) do
```

Comment:
This actually takes env_secret_id according to the controller.

```suggestion
  def delete_env_secret(env_secret_id, params) do
```